docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -0,0 +1,572 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Google Generative Language Semantic Retriever\n",
```

Comment:
pls add a colab link on the top as well. You can do this way:
```
<a href="https://colab.research.google.com/github/run-llama/llama_index/blob/main/docs/examples/vector_stores/SimpleIndexDemo.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
```
replace docs/examples/... with the path to the notebook

Reply:
Added with the suggested change to the path.